e2e/scaling/scaling_test.go

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package scaling
+
+import (
+	"testing"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/nomad/e2e/e2eutil"
+	"github.com/hashicorp/nomad/helper/pointer"
+	"github.com/hashicorp/nomad/helper/uuid"
+	"github.com/hashicorp/nomad/nomad/structs"
+	"github.com/shoenig/test/must"
+	"github.com/shoenig/test/wait"
+)
+
+const defaultNS = "default"
+
+func TestScaling(t *testing.T) {
+
+	nomad := e2eutil.NomadClient(t)
+	e2eutil.WaitForLeader(t, nomad)
+	e2eutil.WaitForNodesReady(t, nomad, 1)
+
+	// Run our test cases.
+	t.Run("TestScaling_Basic", testScalingBasic)
+	t.Run("TestScaling_Namespaces", testScalingNamespaces)
+	t.Run("TestScaling_System", testScalingSystemJob)
+}
+
+func testScalingBasic(t *testing.T) {
+	nomad := e2eutil.NomadClient(t)
+
+	jobID := "scaling-basic-" + uuid.Short()
+	jobIDs := []string{jobID}
+	t.Cleanup(e2eutil.MaybeCleanupJobsAndGC(&jobIDs))
```

Comment:
I would suggest `jobs3` similar to `cluster3` above, but I see now that the refactor is pretty minimal, and switching everything over to e2e/v3 would be a more significant effort.

but it is nice that `jobs3.Submit` does its own cleanup, etc :shrug:

Reply:
Yeah I didn't want to rewrite the tests to that level just to get them off the framework.